From 488ca21e8f7b3704fc3252732da8a0d188cbc458 Mon Sep 17 00:00:00 2001
From: Josh Triplett <josh@joshtriplett.org>
Date: Sun, 22 Apr 2012 20:14:26 -0700
Subject: [PATCH 3/4] In /etc/default/cron, change the default log level to -L
 35, to ensure that cron job output gets logged on
 systems without an MTA.

---
 debian/changelog    |    2 ++
 debian/cron.default |    2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 6f82878..d33c247 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -10,6 +10,8 @@ cron (3.0pl1-122) UNRELEASED; urgency=low
   [ Josh Triplett ]
   * Add two new log levels: -L 16 logs all cron job output, and -L 32 logs all
     cron job output when not already mailing that output.
+  * In /etc/default/cron, change the default log level to -L 35, to ensure
+    that cron job output gets logged on systems without an MTA.
 
  -- Javier Fernández-Sanguino Peña <jfs@debian.org>  Mon, 09 Apr 2012 17:23:01 +0200
 
diff --git a/debian/cron.default b/debian/cron.default
index 92845f9..08d2439 100644
--- a/debian/cron.default
+++ b/debian/cron.default
@@ -26,7 +26,7 @@ READ_ENV="yes"
 #   16  log the full output of all cron jobs even when also mailing that output
 #   32  log the full output of all cron jobs when not mailing that output
 #
-#EXTRA_OPTS=""
+EXTRA_OPTS="-L 35"
 
 # Uncomment to disable lost+found check
 #CHECK_LOSTFOUND=no
-- 
1.7.10

